### PR TITLE
Update for zope-product

### PR DESCRIPTION
**Pull request by icemac**

### CODE REVIEW

**Comment by icemac** [comment]
This PR is needed to get GHA green again after their problems.

File: tox.ini [hunk 1]
```diff
@@ -18,6 +18,7 @@ skip_install = true
 deps =
     setuptools < 52
     zc.buildout
+    zope.testrunner
```
Comment (jugmac00) [inline tox.ini:21]: How did this work before?

File: tox.ini [hunk 2]
```diff
@@ -46,7 +47,6 @@ deps =
     # Useful flake8 plugins that are Python and Plone specific:
     flake8-coding
     flake8-debugger
-    flake8-string-format
```
Comment (jugmac00) [inline tox.ini:49]: Is this intentionally left out?

**Review by jugmac00 — APPROVED** [review]
Just for your info - it is not necessary to create a PR in order to re-run tests, you can just hit the button:

Screenshot from 2021-05-17 10-13-09

It does not hurt either :-)

LGTM with a couple of annotations

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -4,14 +4,17 @@ name: tests
 
 on:
   push:
-    branches: [ master ]
   pull_request:
   schedule:
     - cron: '0 12 * * 0'  # run once a week on Sunday
+  # Allow to run this workflow manually from the Actions tab
+  workflow_dispatch:
```
Comment (icemac) [inline .github/workflows/tests.yml:11]: @jugmac00 This line was needed to be able to re-run the job via the web-UI. It was missing so I was not able to rerun CI without creating a new commit.

File: tox.ini (hunk 1 above)
Comment (icemac) [reply tox.ini:21]: Probably because `zope.testrunner` is part of the `tests` extra.

File: tox.ini (hunk 2 above)
Comment (icemac) [reply tox.ini:49]: Yes, it creates too much hassle to change all the strings so they match the wishes of this checker.

File: .github/workflows/tests.yml (hunk 3 above)
Comment (icemac) [reply .github/workflows/tests.yml:11]: @jugmac00 Interesting you found a hidden switch to re-run the tests without this line.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Comment by jugmac00** [comment]
Thanks for creating this PR and taking the time to answer my questions :)

**Pull request merged**